src/titles.json

```diff
@@ -72,7 +100,12 @@
         "nex_datastore_version": "3.2.1",
         "nex_match_making_version": "3.2.1",
         "nex_messaging_version": "3.2.1",
-        "nex_utility_version": "3.2.1"
+        "nex_utility_version": "3.2.1",
+        "title_ids": [
+            "0005000010113100",
+            "0005000C10113100",
+            "0005000E10113100"
```

Comment:
Commenting on this specific one because it has all 3. We don't need the update and DLC title IDs, only the base title ID. There's a number of titles here which have the update and DLC title IDs listed

Reply:
Fixed in latest commit